meson_options.txt

```diff
@@ -252,5 +252,7 @@ option('zshcompletiondir', type : 'string',
 
 option('tests', type : 'combo', choices : ['true', 'unsafe'],
        description : 'enable extra tests with =unsafe')
+option('slow-tests', type : 'boolean', value : 'false',
+       description : 'run the slow tests by default')
```

Comment:
I don't think that those "slow" tests bring that much value ;) It's good that we can hammer the hashmap code, but I think it'll be enough if we just do that on some fraction of builds — if it were to find any issues, that'll be enough. Similarly for the other "slow" tests. And it's good to make the common case fast, just to encourage users and developers to run the test suite often.